core/src/main/kotlin/fr/sncf/osrd/api/etcs/ETCSBrakingCurvesEndpoint.kt

```diff
@@ -168,6 +223,21 @@ class ETCSBrakingCurvesEndpoint(
         return Envelope.make(*mrspParts.toTypedArray())
     }
 
+    private fun buildETCSConflictCurves(
+        eoaBrakingCurves: EOABrakingCurves
+    ): List<ETCSConflictCurves> {
+        return eoaBrakingCurves.map {
+            assert(it.key.eoaType == EoaType.SPACING || it.key.eoaType == EoaType.ROUTING)
+            ETCSConflictCurves(
+                buildSimpleEnvelope(it.value[BrakingType.IND]!!.brakingCurve)!!,
```

Comment:
Nit and mildly unrelated to this PR

Lots of `!!` there. The last one is because `buildSimpleEnvelope` can accept and return null values, we could instead make it strictly non-null. If we make it an extension method of `Envelope` we could use the `envelope?.buildSimpleEnvelope()` syntax in the few places where it may actually be null.